composer.json

```diff
@@ -25,20 +25,22 @@
         }
     },
     "require": {
-        "php": "^7.2 || ^8.0"
+        "php": "^7.4 || ^8.0"
     },
     "require-dev": {
         "ext-sqlite3": "*",
         "doctrine/doctrine-bundle": "^1.12|~2.0",
         "doctrine/orm": "^2.7",
         "friendsofphp/php-cs-fixer": "^2.2",
-        "phpdocumentor/reflection-docblock": "^4.0|~5.0",
+        "phpdocumentor/reflection-docblock": "^4.4|^5.0",
         "phpspec/prophecy": "^1.7",
-        "symfony/framework-bundle": "^3.4.25|~4.0",
-        "symfony/mime": "^4.4.|~5.0",
-        "symfony/phpunit-bridge": "~5.0",
-        "symfony/twig-bundle": "^3.4|~4.0",
-        "symfony/yaml": "^3.4|~4.0"
+        "phpspec/prophecy-phpunit": "^2.0",
+        "phpunit/phpunit": "^9.5",
+        "symfony/framework-bundle": "^4.4|^5.0|^6.0",
+        "symfony/mime": "^4.4.|^5.0|^6.0",
+        "symfony/phpunit-bridge": "^6.0",
```

Comment:
Is it still needed, since 

1. you don't use vendor/bin/simple-phpunit 
2. you don't register the symfony listener anymore 

?